spec/jobs/pull_hotlinked_images_spec.rb

```diff
@@ -29,4 +29,42 @@
     expect(post.raw).to match(/^<img src='\/uploads/)
   end
 
+  describe 'onebox' do
+
+    Onebox::Engine::WikimediaOnebox.class_eval do
```

Comment:
Yes. It may be possible. But that looks difficult to me. And we already using this method in many places at `onebox` gem source.

https://github.com/discourse/onebox/blob/master/spec/lib/onebox/engine/whitelisted_generic_onebox_spec.rb#L33
https://github.com/discourse/onebox/blob/master/spec/lib/onebox/engine_spec.rb#L11
https://github.com/discourse/onebox/blob/master/spec/lib/onebox/layout_spec.rb#L52